src/lib/syncDocsPath.ts

```diff
@@ -127,6 +130,40 @@ async function pushDoc(
     });
 }
 
+function sortFiles(files: string[], { allowedFileExtensions }: { allowedFileExtensions: string[] }): string[] {
```

Comment:
this function appears to use `parentDocSlug`, is there any way we add basic support for `parentDoc`? i know it uses object IDs so there's no way for us to build a graph, but at the very least we could make it so docs that have a `parentDoc` attribute get synced after ones that lack one?